OMCompiler/SimulationRuntime/c/util/utility.c

```diff
@@ -160,32 +160,12 @@ void OpenModelica_updateUriMapping(threadData_t *threadData, void *namesAndDirs)
 
 static const char *PATH_NOT_IN_FMU_RESOURCES = "Returning path (%s) not in the resources directory. The FMU might not work as expected if you send it to a different system";
 
-static int isWindows(void)
+/* if uri starts with X:\ or X:/ */
+static int hasDriveLetter(const char* uri)
 {
-#if defined(__MINGW32__) || defined(_MSC_VER)
-  return 1;
-#else /* not windows */
-  return 0;
-#endif
-}
-
-static int isMinGW(void)
-{
-#if defined(__MINGW32__)
-  return 1;
-#else /* not MinGW */
-  return 0;
-#endif
-}
-
-static int isDriveLetter(char c)
-{
-  return isWindows() && ((c >= 'A' && c <= 'Z') || (c >= 'a' && c <= 'z'));
-}
-
-static int isPathSeparator(char c)
-{
-  return (c == '/') || (isWindows() && (c == '\\'));
+  return strlen(uri) > 2 &&
```

Comment:
Maybe have 2 as a constant or #define so we know why we have the value 2?

Reply:
Is 2 because it needs to have at least X:\ in it, if is less then it cannot be a drive letter :)